crypto/codec/proto.go

```diff
@@ -16,6 +17,7 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	registry.RegisterImplementations(pk, &ed25519.PubKey{})
 	registry.RegisterImplementations(pk, &secp256k1.PubKey{})
 	registry.RegisterImplementations(pk, &multisig.LegacyAminoPubKey{})
+	registry.RegisterImplementations(pk, &unclaimable.PubKey{})
```

Comment:
Do I need amino registration here?

Reply:
yes, let's add amino

Reply:
Alright, added. Not sure about the name.